framework/src/modules/token/commands/transfer.ts

```diff
@@ -56,6 +97,14 @@ export class TransferCommand extends BaseCommand {
 
 	public async execute(context: CommandExecuteContext<Params>): Promise<void> {
 		const { params } = context;
+
+		await this._intializeUserStore(
+			context,
+			params.recipientAddress,
+			params.tokenID,
+			context.transaction.senderAddress,
+		);
+
```

Comment:
Now `transferInternal` has bit different logic than `transfer`, maybe you can create an empty function `transferInternal` on `method.ts` with the same interface and call the function for now